README.md

```diff
@@ -47,6 +48,10 @@ To build a VM machine from this repo's source, see the [instructions](docs/creat
 
 There are no migrations in progress.
 
+## Announcements
+
+You can also view notable upcoming changes by viewing the [Announcement](https://github.com/actions/runner-images/labels/Announcement) label.
+

```

Comment:
Wondering if we should rename `Ongoing migrations` to `Announcements` and when we have a migration we can simply make an announcement issue for it. wdyt?

```suggestion
## Announcements

See notable upcoming changes by viewing the issues with the [Announcement](https://github.com/actions/runner-images/labels/Announcement) label.


```

Reply:
@samvantran works for me